book/C-git-commands/1-git-commands.asc

```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
+Ang `git tag` na utos ay ginamit para mabigay ng permanenteng bookmark para sa tiyak na punto sa kasaysayan ng code.
 
-We also cover how to create a GPG signed tag with the `-s` flag and verify one with the `-v` flag in <<_git_tools#_signing>>.
+Ang utos na ito ay pinakilala at tinalakay ng detalye sa <<_git_basics_chapter#_git_tagging>> at ginamit namin ito sa pagsasanay sa <<_distributed_git#_tagging_releases>>.
 
+Tinalakay din namain kung paano gumawa ng naka tag na sign na GPG sa pamamagitan ng `--s` na flag at pinatunayan ang isa sa pamamagitan ng `-v` na flag sa <<_git_tools#_signing>>.
 
-=== Sharing and Updating Projects
+=== Pagbabahagi at Pagbabago ng mga Proyekto
 
-There are not very many commands in Git that access the network, nearly all of the commands operate on the local database.
-When you are ready to share your work or pull changes from elsewhere, there are a handful of commands that deal with remote repositories.
+Walang masyadong maraming mga utos sa Git na may access sa network, halos ang lahat ng mga utos ay gumagana sa lokal na database. Kung ikaw ay handa ng ibahagi ang iyong trabaho o mag-pull ng pagbabago galing sa ibang lugar, mayroon kakaunting mga utos na tumatalakay sa malayong mga repositoryo.
 
 ==== git fetch
 
-The `git fetch` command communicates with a remote repository and fetches down all the information that is in that repository that is not in your current one and stores it in your local database.
+Ang `git fetch` na utos ay nakikipag-usap sa malayong repositoryo at kumukuha ng lahat ng impormasyon na nasa loob ng repositoryo na hindi kasali sa iyong kasalukuyan at iniimbak ito sa iyong lokal na database.
 
-We first look at this command in <<_git_basics_chapter#_fetching_and_pulling>> and we continue to see examples of it use in <<_git_branching#_remote_branches>>.
+Una naming tiningnan ang utos na ito sa <<_git_basics_chapter#_fetching_and_pulling>> at patuloy nating makikita ang mga halimbawa sa paggamit nito sa <<_git_branching#_remote_branches>>.
 
-We also use it in several of the examples in <<_distributed_git#_contributing_project>>.
+Ginamit din namin ito sa iilang mga halimbawa sa <<_distributed_git#_contributing_project>>.
 
-We use it to fetch a single specific reference that is outside of the default space in <<_github#_pr_refs>> and we see how to fetch from a bundle in <<_git_tools#_bundling>>.
+Ginamit namin ito para kumuha ng nag-iisang tiyak na sanggunian na nasa labas ng default na lugar sa <<_github#_pr_refs>> at nakita din natin kung paano kunin ang nanggaling sa bigkis sa <<_git_tools#_bundling>>.
 
-We set up highly custom refspecs in order to make `git fetch` do something a little different than the default in <<_git_internals#_refspec>>.
+Nagset-up kami ng napakainam na custom refspecs para lang makagawa ang `git fetch` ng bagay na medyo naiiba kaysa sa sanggunian sa <<_git_internals#_refspec>>.
```

Comment:
``para lang makagawa ang `git fetch` ``  to `` para lang makagawa ng `git fetch` ``